src/erlcloud_cloudtrail.erl

```diff
@@ -115,6 +115,14 @@ describe_trails(Trails, Config) ->
     Json = [{<<"TrailNameList">>, [list_to_binary(T) || T <- Trails]}],
     ct_request("DescribeTrails", Json, Config).
 
+-spec describe_trails([string()], aws_config(), boolean()) -> ct_return().
+describe_trails([], Config, IncludeShadowTrails)
+        when IncludeShadowTrails =:= false ->
```

Comment:
IMO this guard/function clause is unnecessary and simple guard for boolean is enough.
```erlang
describe_trails([], IncludeShadowTrails, Config) when is_boolean(IncludeShadowTrails) ->
    Json = [{<<"includeShadowTrails">>, IncludeShadowTrails}],
    ct_request("DescribeTrails", Json, Config).

Reply:
@motobob and what if IncludeShadowTrails is false?